windows/src/developer/TIKE/main/UframeTextEditor.pas

```diff
@@ -231,12 +231,11 @@ procedure TframeTextEditor.ReplaceSelection(ARange: TRect;
   end;
 end;
 
-procedure TframeTextEditor.cefBeforeBrowse(Sender: TObject; const Url: string; params: TStringList; wasHandled: Boolean);
+procedure TframeTextEditor.cefCommand(Sender: TObject; const command: string; params: TStringList);
 begin
   AssertVclThread;
-  if (params.Count > 0) and (params[0] = 'command') then
+  if command = 'command' then
   begin
-    params.Delete(0);
     FireCommand(params);
```

Comment:
Do you still want to check if `params.Count > 0`?

Reply:
Same question for `TframeTouchLayoutBuilder.cefCommand()`

Reply:
Now that the command is passed as a parameter, there is no need to any more as the test was merely to make sure we didn't get an array out of bounds error.